arithmetization/src/main/java/net/consensys/linea/zktracer/module/romlex/RomLex.java

```diff
@@ -189,7 +187,11 @@ public void callRomLex(final MessageFrame frame) {
         checkArgument(length > 0, "callRomLex for RETURN expects positive size");
 
         byteCode = frame.shadowReadMemory(offset, length);
-        hub.defers().scheduleForContextExit(this, hub.currentFrame().id());
+        Address deploymentAddress = hub.currentFrame().byteCodeAddress();
+        final ContractMetadata contractMetadata =
+                ContractMetadata.make(deploymentAddress, hub.deploymentNumberOf(deploymentAddress), false);
+        final RomOperation chunk = new RomOperation(contractMetadata, false, true, byteCode);
+        operations.add(chunk);
```

Comment:
Why were we scheduling production of the contract metadata etc ? We have all the information required.